flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/core/testutils/OneShotLatch.java

```diff
@@ -61,7 +61,11 @@ public void await() throws InterruptedException {
 				try {
 					waitersSet.add(thread);
 					lock.wait();
-				} finally {
+				}
+				catch (InterruptedException e) {
+					Thread.currentThread().interrupt();
+				}
+				finally {
```

Comment:
I don't understand what this change will do for fixing the `SystemProcessingTimeServiceTest` instability.

Reply:
hi ,@tillrohrmann.I'm not sure if I can solve the instability with the change,but it does exist.

Reply:
Without an explanation why this solves the problem, I cannot merge this PR.

Reply:
ok,thanks.I will spend more time researching this issue

Reply:
Ok, then let's close this PR for the time being.